frontend/public/components/RBAC/bindings.jsx

```diff
@@ -48,7 +48,7 @@ export const flatten = resources => _.flatMap(resources, resource => {
       ret.push(Object.assign({}, binding, {
         subject,
         subjectIndex,
-        rowKey: `${getQN(binding)}|${subject.kind}|${subject.name}`,
+        rowKey: `${getQN(binding)}|${subject.kind}|${subject.name}${typeof subject.namespace !== 'undefined' && `|${subject.namespace}`}`,
```

Comment:
nit: this will append `false` to the string when namespace isn't set. Unlikely to be a problem in practice, but technically not guaranteed to be unique and probably not intended.

```suggestion
        rowKey: `${getQN(binding)}|${subject.kind}|${subject.name}${subject.namespace ? `|${subject.namespace}` : ''}`,
```